### DISCUSSION TITLE
MSVC /MT mode compilation failed

### DISCUSSION THREAD

**Discussion by tsxiaofang** [fltk-rs/fltk-rs#1364] [General]
The default compilation parameter of msvc is /MD. How to switch to /MT.

**Comment by MoAlyousef** [comment]
Some CMake flags are overridden by the cmake crate which fltk-rs uses. /MT can be supported, but you'll have to create a CMake toolchain file with `add_compile_options(/MT)`, and pass it via CFLTK_TOOLCHAIN env variable. Otherwise you can build cfltk from source using CMake with /MT, install it, then point to the libraries generated by it using CFLTK_BUNDLE_DIR env variable.

**Reply by MoAlyousef** [reply]
I've added support for it. You can test the git repo while using the crt-static flag.
